src/main/java/com/es/phoneshop/web/ProductListPageServlet.java

```diff
@@ -18,13 +22,23 @@
 	public void init(ServletConfig config) throws ServletException{
 		
 		super.init(config);
-		productDao = new ArrayListProductDao();
+		productDao = ArrayListProductDao.getInstance();
 		
 	}
 	
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-        request.setAttribute("products", productDao.findProducts());
+    	//Constants
+        String field = "field";
+        String order = "order";
```

Comment:
Move this to constants on a class level. Otherwise they will be created each time doGet is called